.github/CODEOWNERS

```diff
@@ -5,4 +5,4 @@
 
 
 # Fallback on any other change if no other pattern matches below.
-*                   taylan@edgedelta.com fatih@edgedelta.com ozgur@edgedelta.com orhan@edgedelta.com tuncer@edgedelta.com veysi@edgedelta.com gozde@edgedelta.com burak@edgedelta.com asim@edgedelta.com latif@edgedelta.com
\ No newline at end of file
+*                   fatih@edgedelta.com ozgur@edgedelta.com orhan@edgedelta.com tuncer@edgedelta.com veysi@edgedelta.com gozde@edgedelta.com burak@edgedelta.com asim@edgedelta.com
```

Comment:
You can also add @yterradas here

Reply:
Added thank you

Reply:
may I also request to add me? I can also open another PR if you mind

Reply:
Done, thank you